bindings/core/src/method_handler/utils.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_sdk::{
+    client::{hex_public_key_to_bech32_address, hex_to_bech32, request_funds_from_faucet, verify_mnemonic, Client},
+    types::block::{
+        address::{dto::AddressDto, Address, Ed25519Address},
+        output::{AliasId, FoundryId, NftId},
+        payload::{transaction::TransactionEssence, TransactionPayload},
+        signature::Ed25519Signature,
+        Block,
+    },
+};
+use zeroize::Zeroize;
+
+use crate::{method::UtilsMethod, method_handler::Result, response::Response};
+
+/// Call a utils method.
+pub(crate) async fn call_utils_method_internal(method: UtilsMethod) -> Result<Response> {
+    match method {
+        UtilsMethod::Bech32ToHex { bech32 } => Ok(Response::Bech32ToHex(Client::bech32_to_hex(&bech32)?)),
```

Comment:
Feels inconsistent to call `bech32_to_hex` through wallet but not `hex_to_bech32`. Since it's utils, I wouldn't go through the client